ouroboros-consensus/src/Ouroboros/Consensus/Node/Recovery.hs

```diff
@@ -1,13 +1,21 @@
-{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE AllowAmbiguousTypes #-}
```

Comment:
Same remark as above. In fact, `AllowAmbiguousTypes` is a pragma non grata in the consensus :slightly_smiling_face: 

I found only one use of it in master and that can and should be removed.